coderd/notifications/notiffake/notiffake.go

```diff
@@ -0,0 +1,37 @@
+package notiffake
```

Comment:
Nit: I like splitting this out but I think the name is too "cute".
I think something like `testutil` or something would be more idiomatic.

Reply:
Alright, I decided to merge it as is, and refactor/unify once https://github.com/coder/coder/pull/13868 is merged.